src/shared/codelens/localLambdaRunner.ts

```diff
@@ -49,7 +49,7 @@ export interface OnDidSamBuildParams {
     isDebug: boolean
 }
 
-const TEMPLATE_RESOURCE_NAME: string = 'awsToolkitSamLocalResource'
+const TEMPLATE_RESOURCE_NAME = 'awsToolkitSamLocalResource'
```

Comment:
This way, its type is `'awsToolkitSamLocalResource'` instead of `string`. The type (string or string literal) doesn't particular matter in this case, but it's nice to be able to mouse-over `TEMPLATE_RESOURCE_NAME` and see its value.